<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="address_name">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name of person to add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_cell">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Cellphone of person to add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_email">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Email of the person to add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="append">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Append to list</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <fontweight>Light</fontweight>
     </font>
    </property>
    <property name="text">
     <string>Address Book</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove from list</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Create a row by typing in the fields and clicking on the associated button</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <fontweight>Light</fontweight>
     </font>
    </property>
    <property name="text">
     <string>How it works</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Double click on a row to edit it</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3. Select a row and click on &quot;Remove from list&quot; to remove it</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
